webui/react/src/components/TimeAgo.tsx

```diff
@@ -63,6 +63,25 @@ const TimeAgo: React.FC<Props> = ({
     }
     return undefined;
   }, [datetime]);
+  const tooltip: ReactNode = useMemo(() => {
+    if (tooltipFormat === DEFAULT_TOOLTIP_FORMAT) return tooltipFormat;
+
+    let base = dayjs(milliseconds).format(tooltipFormat);
+    // Removes a few possible formats of dates from the end of the string to be added after a line break.
+    const check = base
+      .split(/\w{3,} \d{1,2}, \d{2,4}( - \d{1,2}:\d{1,2}( \w{2}|\s)|\s)$/)[0]
```

Comment:
this code only works with a specific format. If input format changes, we need to change it here. 
How about this? This isnt the best but the change is minimal and it works with any formats
```diff
diff --git a/webui/react/src/components/ProjectCard.tsx b/webui/react/src/components/ProjectCard.tsx
index d4729e7a1..c3e4d7b4f 100644
--- a/webui/react/src/components/ProjectCard.tsx
+++ b/webui/react/src/components/ProjectCard.tsx
@@ -81,7 +81,7 @@ const ProjectCard: React.FC<Props> = ({
                 <TimeAgo
                   className={css.lastExperiment}
                   datetime={project.lastExperimentStartedAt}
-                  tooltipFormat="[Last experiment started ]MMM D, YYYY - h:mm a"
+                  tooltipFormat="[Last experiment started \n]MMM D, YYYY - h:mm a"
                 />
               )
             )}
diff --git a/webui/react/src/components/TimeAgo.tsx b/webui/react/src/components/TimeAgo.tsx
index 1d12e7e2b..99af35bf5 100644
--- a/webui/react/src/components/TimeAgo.tsx
+++ b/webui/react/src/components/TimeAgo.tsx
@@ -116,7 +116,12 @@ const TimeAgo: React.FC<Props> = ({
   }, [updateInterval]);
 
   return (
-    <Tooltip content={dayjs(milliseconds).format(tooltipFormat)}>
+    <Tooltip
+      content={
+        <div style={{ whiteSpace: 'pre-wrap' }}>
+          {dayjs(milliseconds).format(tooltipFormat).replace('\\n', '\n')}
+        </div>
+      }>
       <div className={classes.join(' ')}>{durationString}</div>
     </Tooltip>
   );
```